crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -47,7 +47,9 @@ def restore_chef
               if proposal?(filename) && type == :databags
                 bc_name, prop = filename.split("-")
                 prop.gsub!(/.json$/, "")
-                Proposal.create(barclamp: bc_name, name: prop, properties: record.raw_data)
+                proposal = Proposal.first_or_initialize(barclamp: bc_name, name: prop)
```

Comment:
It needs to be:

```Proposal.where(barclamp: bc_name).first_or_initialize(barclamp: bc_name, name: prop)```

Otherwise we will just overwrite the first Proposal (which ever that is) again and again.